### DISCUSSION TITLE
Not working on Mac M1

### DISCUSSION THREAD

**Discussion by changkhothuychung** [utmapp/UTM#5334] [Linux]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
When running an Ubuntu image on Mac M1, I get "display is not initialized yet", does anyone know how to fix this isue?

**Comment by Tropical-Smootie** [comment]
what version UTM
and macOS are you running?
also edit the post, delete the unwanted stuff.
https://youtu.be/82OFWDyCs38 , this video might help
 refer to this issue #5066

**Comment by changkhothuychung** [comment]
UTM version is 4.2.5 and MacOS is Ventura 13.2.1, I want to install Ubuntu. 

Generally, does UTM support mac M1 ?

**Comment by Tropical-Smootie** [comment]
Yes UTM is made specifically for Apple Silicon, M1 and M2.
